actions/test/run_local.ts

```diff
@@ -52,9 +52,10 @@ const main = async () => {
       `[run_local] Processing ONCE for a specific transaction: ${txEnv}...`
     );
 
-    await processTx(provider, txEnv, chainId, testRuntime).catch(() =>
-      exit(ERROR_CODE_PROCESS_TX)
-    );
+    await processTx(provider, txEnv, chainId, testRuntime).catch((e) => {
+      console.error(e);
+      exit(ERROR_CODE_PROCESS_TX);
+    });
```

Comment:
Can you use a try catch instead?

Reply:
agree is not the best to finish execution in the middle. This part is being redone in the dockerised version.
I think is not worth it to through effort in this branch. We will need to improve a lot the standalone one.